### PR TITLE
Remove reconfigure-kubelet page

### LINKED ISSUE #31585: Remove page “Reconfigure a Node's Kubelet in a Live Cluster”
**This is a Cleanup Request**

**What would you like to be removed**
Remove https://kubernetes.io/docs/tasks/administer-cluster/reconfigure-kubelet/

**Why is this needed**
v1.24 of Kubernetes will remove the dynamic kubelet config feature (deprecated since Kubernetes v1.22)

**Comments**
/sig node
/language en
/milestone 1.24

### PR DESCRIPTION
**Pull request by mboersma**
This change removes the "Reconfigure a Node's Kubelet in a Live Cluster" page and internal references to it.

Fixes #31585

### CODE REVIEW

File: content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ been deprecated. These removals have been superseded by newer, stable/generally
 
 ## API removals, deprecations, and other changes for Kubernetes 1.24
 
-* [Dynamic kubelet configuration](https://github.com/kubernetes/enhancements/issues/281): `DynamicKubeletConfig` is used to enable the dynamic configuration of the kubelet. The `DynamicKubeletConfig` flag was deprecated in Kubernetes 1.22. In v1.24, this feature gate will be removed from the kubelet. See [Reconfigure kubelet](/docs/tasks/administer-cluster/reconfigure-kubelet/). Refer to the ["Dynamic kubelet config is removed" KEP](https://github.com/kubernetes/enhancements/issues/281) for more information. 
+* [Dynamic kubelet configuration](https://github.com/kubernetes/enhancements/issues/281): `DynamicKubeletConfig` is used to enable the dynamic configuration of the kubelet. The `DynamicKubeletConfig` flag was deprecated in Kubernetes 1.22. In v1.24, this feature gate was removed from the kubelet. Refer to the ["Dynamic kubelet config is removed" KEP](https://github.com/kubernetes/enhancements/issues/281) for more information.
```
Comment (sftim) [inline content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:70]:
```suggestion
* [Dynamic kubelet configuration](https://github.com/kubernetes/enhancements/issues/281): `DynamicKubeletConfig` is used to enable the dynamic configuration of the kubelet. The `DynamicKubeletConfig` flag was deprecated in Kubernetes 1.22. In v1.24, this feature gate will be removed from the kubelet. Refer to the ["Dynamic kubelet config is removed" KEP](https://github.com/kubernetes/enhancements/issues/281) for more information.
```

Blog articles are written as if they are applicable to their publication date.

**Review by sftim — COMMENTED** [review]
LGTM, with a caveat.

**Review by dipesh-rawat — COMMENTED** [review]
The deletion of the page and its corresponding links from all other places looks good.

@mboersma It appears that there is still one remaining [page](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/feature-gates-removed.md?plain=1#L571) that contains a link to the removed page. Would it be possible for you to remove that link as well as part of this PR?
_Page left to edit: [feature-gates-removed.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/feature-gates-removed.md?plain=1#L571)_

**Comment by mboersma** [comment]
> there is still one remaining [page](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/feature-gates-removed.md?plain=1#L571) that contains a link to the removed page

Fixed, thank you for catching that!

I think the only remaining non-localized reference is in the blog post specifically about dynamic kubelet configuration: https://github.com/kubernetes/website/blob/main/content/en/blog/_posts/2018-07-11-dynamic-kubelet-configuration.md?plain=1#L50

Removing the link implies removing the entire `How can I learn more?` section really, and I wasn't sure whether to do that, edit it to say that this feature is removed, or leave it as-is.

**Pull request merged**